integration-tests/common/common.go

```diff
@@ -115,78 +124,55 @@ func stripKeyPrefix(key string) string {
 	return key
 }
 
-func New(env string, isK8s bool) *Common {
-	var err error
+func New(testConfig *tc.TestConfig) *Common {
 	var c *Common
-	if env == "devnet" {
-		c = &Common{
-			IsK8s:     isK8s,
-			ChainName: ChainName,
-			ChainId:   DevnetChainID,
-			SolanaUrl: SolanaDevnetURL,
-		}
-	} else {
-		c = &Common{
-			IsK8s:     isK8s,
-			ChainName: ChainName,
-			ChainId:   LocalnetChainID,
-			SolanaUrl: SolanaLocalNetURL,
-		}
+
+	// Setting localnet as the default config
+	config := chainConfig.LocalNetConfig()
+	// Getting the default localnet private key
+	privateKey, err := solana.PrivateKeyFromBase58(solclient.DefaultPrivateKeysSolValidator[1])
+	if err != nil {
+		panic("Could not decode private devnet private key")
```

Comment:
should this say localnet or devnet? it looks like it's referencing localnet?

or maybe it's better to not specify network if it's not specific to localnet or devnet